<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setActiveProfilesDlg</class>
 <widget class="QDialog" name="setActiveProfilesDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Active Profiles</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="setProjectsBtn">
     <property name="text">
      <string>Set Active Projects</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="setProfilesBtn">
     <property name="text">
      <string>Set Active Profiles</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="setSpcCollectionsBtn">
     <property name="text">
      <string>Set Active Species Collections</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="setMetaGroupsBtn">
     <property name="text">
      <string>Set Active Metadata Groups</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
